cmd/protoc-gen-docs/htmlGenerator.go

```diff
@@ -548,37 +587,39 @@ func (g *htmlGenerator) generateMessage(message *protomodel.MessageDescriptor) {
 					}
 				}
 
+				var required bool = false
+				if field.Options != nil {
+					fb := getFieldBehavior(field.Options)
+					for _, e := range fb {
+						if e == annotations.FieldBehavior_REQUIRED {
+							required = true
+						}
+					}
+				}
+
 				id := normalizeID(g.relativeName(field))
 				if class != "" {
 					g.emit(`<tr id="`, id, `" class="`, class, `">`)
 				} else {
 					g.emit(`<tr id="`, id, `">`)
 				}
 				fieldLink := `<a href="#` + id + "\">" + fieldName + "</a>"
-				g.emit("<td><code>", fieldLink, "</code></td>")
-				g.emit("<td><code>", g.linkify(field.FieldType, fieldTypeName, true), "</code></td>")
+
+				// field
+				g.emit("<td><code>", fieldLink, "</code><br/>")
+				// type
+				g.emit("<em>", g.linkify(field.FieldType, fieldTypeName, true), "</em><br/>")
+				// required
+				if required {
+					g.emit("Required")
+				} else {
+					g.emit("Optional")
```

Comment:
nit: I would just put 'required' when its required and omit it otherwise? The 3 line colunm is slightly noisy so that would clean it up

Reply:
Could also put it as the first word of the description field?

Reply:
Done, PTAL. 
A more correct thing to do would be to add CSS classes, and to reduce the font size of the type in the left hand column.